PKHeX.Core.AutoMod/AutoMod/APILegality.cs

```diff
@@ -894,9 +900,16 @@ private static void FindTeraPIDIV(PK9 pk, EncounterTera9 enc, IBattleTemplate se
                 const byte rollCount = 1;
                 const byte undefinedSize = 0;
                 var pi = PersonalTable.SV.GetFormEntry(pk.Species, pk.Form);
```

Comment:
Will have issues for gender-ratio changed species like Azurill or Indeedee (from Espurr). Needs to use the enc Species/Form.